### PR TITLE
feat: [STO-4486]: STO Project setup menu

### PR DESCRIPTION
**Pull request by jsentongo**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by jsentongo** [comment]
retrigger jest

File: src/modules/10-common/navigation/ProjectSetupMenu/ProjectSetupMenu.tsx [hunk 1]
```diff
@@ -48,16 +48,14 @@ const ProjectSetupMenu: React.FC<ProjectSetupMenuProps> = ({ module }) => {
...
           <SidebarLink label={getString('common.templates')} to={routes.toTemplates(params)} />
         )}
-        {CVNG_TEMPLATE_MONITORED_SERVICE && isCV && (
-          <SidebarLink
-            label={getString('common.templates')}
-            to={routes.toTemplates({ ...params, templateType: 'MonitoredService' })}
-          />
-        )}
-        {OPA_PIPELINE_GOVERNANCE && isCIorCD && canUsePolicyEngine && (
+        {OPA_PIPELINE_GOVERNANCE && isCIorCDorSTO && canUsePolicyEngine && (
```
Comment (chris-harness) [inline src/modules/10-common/navigation/ProjectSetupMenu/ProjectSetupMenu.tsx:58]: This doesn't look right. Why did `isCV` go away?

File: src/modules/10-common/navigation/ProjectSetupMenu/ProjectSetupMenu.tsx [hunk 2]
```diff
@@ -25,14 +25,14 @@ interface ProjectSetupMenuProps {
 const ProjectSetupMenu: React.FC<ProjectSetupMenuProps> = ({ module }) => {
   const { getString } = useStrings()
   const { accountId, orgIdentifier, projectIdentifier } = useParams<PipelineType<ProjectPathProps>>()
-  const { NG_TEMPLATES, OPA_PIPELINE_GOVERNANCE, NG_VARIABLES, CVNG_TEMPLATE_MONITORED_SERVICE } = useFeatureFlags()
+  const { NG_TEMPLATES, OPA_PIPELINE_GOVERNANCE, NG_VARIABLES, NG_GIT_EXPERIENCE } = useFeatureFlags()
```
Comment (chris-harness) [inline src/modules/10-common/navigation/ProjectSetupMenu/ProjectSetupMenu.tsx:28]: This change seems unrelated. Can you make sure this branch isn't mixed up with other changes

**Review by chris-harness — CHANGES_REQUESTED** [review]

**Review by atmollo-harness — APPROVED** [review]

**Review by chris-harness — APPROVED** [review]

File: src/modules/10-common/navigation/ProjectSetupMenu/ProjectSetupMenu.tsx [hunk 3]
```diff
@@ -48,7 +48,7 @@ const ProjectSetupMenu: React.FC<ProjectSetupMenuProps> = ({ module }) => {
             to={routes.toGitSyncAdmin({ accountId, orgIdentifier, projectIdentifier, module })}
           />
         ) : null}
-        {NG_TEMPLATES && isCIorCD && (
+        {NG_TEMPLATES && isCIorCDorSTO && (
```
Comment (abhinavrastogi-harness) [inline src/modules/10-common/navigation/ProjectSetupMenu/ProjectSetupMenu.tsx:51]: Does STO support templates?

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**